content/syllabuses/bootcamp-data-eng-auto-boot.md

```diff
@@ -1,8 +1,10 @@
 ---
 _db_id: 83
 title: Data engineering boot camp (with automarked katas)
+ready: true
```

Comment:
Syllabuses dont need to be marked as ready.  These are not the same as topics or projects.